### DISCUSSION TITLE
csv import issue [May 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2061] [Discord Threads]
> 🙋‍♂️ **MaxWell94**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### hi  
#  
> 🙋‍♂️ **MaxWell94**  
#### hi  
#  
> 👨‍🚀 **wingkwong**  
#### when u import it, can u see if there s any mapping related to mobile?  
#  
> 🙋‍♂️ **MaxWell94**  
#### Thats what I changed that mapping to text  
#  
> 🙋‍♂️ **MaxWell94**  
#### but the error still persists  
#  
> 👨‍🚀 **wingkwong**  
#### can u provide a dummy file for reproducing the issue?  
#  
> 🙋‍♂️ **Bergi9**  
#### Maybe you had that one column as mobile field in the past. Deleting the column and recreate the column should work again.  
#  
> 🙋‍♂️ **MaxWell94**  
#### Yes deleting and recreating it worked.  
#  
> 🙋‍♂️ **MaxWell94**  
#### Thanks guys 🙂  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/972437793349455913)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
